### DISCUSSION TITLE
.sfz

### DISCUSSION THREAD

**Discussion by midimusic** [FluidSynth/fluidsynth#732] [Ideas]
I recently got the impression that .sfz might be an up and coming format for sound font files especially where the highest possible sound quality is desired (as in organs). This format appears to be immature at present but does have future possibilities.

Has anybody considered making fluidsynth able to use these fonts?

Or is there a good reason why fluidsynth can not be adapted to use .sfz sound fonts?

csw900

**Comment by derselbst** [comment]
> Has anybody considered making fluidsynth able to use these fonts?

Yes:
https://lists.nongnu.org/archive/html/fluid-dev/2019-10/msg00004.html
https://lists.nongnu.org/archive/html/fluid-dev/2019-10/msg00005.html

> Or is there a good reason why fluidsynth can not be adapted to use .sfz sound fonts?

Because it's too powerful and doesn't fit into our architecture: https://github.com/FluidSynth/fluidsynth/issues/620#issuecomment-589981458

**Reply by midimusic** [reply]
Thanks for the information - I will investigate your suggestions and continue my experiments with sfz.
csw900
